public/locales/ja-JP/dashboard.json

```
@@ -1000,7 +1001,9 @@
     "driverRootDes": "OneDriveアカウント内でファイルを保存する場所を選択してください。このオプションを変更すると、ストレージポリシーに既に存在するファイルにアクセスできなくなる可能性があります。",
     "saveToDefaultOneDrive": "ファイルをデフォルトのOneDriveドライブに保存",
     "saveToSharePoint": "SharePointにファイルを保存",
-    "sharePointUrlDes": "SharePointサイトのURLを入力してください。フォーカスが外れると、システムが自動的に正しいドライブ識別子に変換します。"
+    "sharePointUrlDes": "SharePointサイトのURLを入力してください。フォーカスが外れると、システムが自動的に正しいドライブ識別子に変換します。",
+    "ks3selectRegionDes": "バケットが存在するリージョンコードを入力してください（例： <0>BEIJING</0>）。",
```

Remove a white space.
```suggestion
    "ks3selectRegionDes": "バケットが存在するリージョンコードを入力してください（例：<0>BEIJING</0>）。",
```